oscap_report/report_parser/report_parser.py

```diff
@@ -0,0 +1,4 @@
+
+class ReportParser():
```

Comment:
Isn't ReportParser a confusing name? I'm aware that in SCAP terminology the word "report" is used for also in ARF ( the R in ARF stands for "reporting"). But when I think of a "report", the first thing that comes to my mind is the HTML report. But you want to parse ARF, or maybe XCCDF results files here. So what about renaming it to "ResultsParser" or "SCAPResultsParser" or similar? Proper naming is difficult.